compiler/rustc_session/src/options.rs

```diff
@@ -1166,6 +1166,8 @@ options! {
     dont_buffer_diagnostics: bool = (false, parse_bool, [UNTRACKED],
         "emit diagnostics rather than buffering (breaks NLL error downgrading, sorting) \
         (default: no)"),
+    drop_tracking: bool = (false, parse_bool, [TRACKED],
```

Comment:
I might call this flag `generator_drop_tracking` to make the scope more clear, although I think it's fine if you want to leave it as is since the help text makes the scope clear.